pagopa/io-ipatente
# [IOPAE-1578,IOPAE-1579] payments summary and detail pages

#### List of Changes

- Adapt summary page to fit payments data
- ListItemAction icon prop became optional
- Generate detail page



#### Motivation and Context



#### How Has This Been Tested?

- Load page /payments and check various status
- Check page /payments/`{requestID}` to check datas





#### Screenshots (if appropriate):


<img width="608" alt="Screenshot 2024-11-21 alle 11 43 01" src="https://github.com/user-attachments/assets/81c5739b-75fb-4a2a-8de6-70e1570412ad">

<img width="608" alt="Screenshot 2024-11-21 alle 11 44 02" src="https://github.com/user-attachments/assets/9288f548-3f00-4c83-9044-3857211f7ebe">

<img width="608" alt="Screenshot 2024-11-21 alle 11 44 30" src="https://github.com/user-attachments/assets/622216c5-c088-42aa-9328-327a5d6306e9">

#### Checklist:
